app/views/home/index.html.erb

```diff
@@ -1,7 +1,3 @@
-<% content_for :banner do %>
-  <%= t('multifactor_auths.setup_webauthn_html') %>
```

Comment:
Is it worth keeping the I18n key?

Reply:
It's still used in the session controller https://github.com/rubygems/rubygems.org/blob/f4be72214042a4c6b7e6c67faa06b3dd2f482cda/app/controllers/sessions_controller.rb#L118 but we're planning to remove that in 2-3 months it was shipped (mid to end of October)